crewai_tools/tools/vision_tool/vision_tool.py

```diff
@@ -64,28 +96,23 @@ def _run(self, **kwargs) -> str:
                 except Exception as e:
                     return f"Error processing image: {str(e)}"
 
-            response = self.client.chat.completions.create(
-                model="gpt-4o-mini",
+            response = self.llm.call(
                 messages=[
-                    {
-                        "role": "user",
-                        "content": [
-                            {"type": "text", "text": "What's in this image?"},
-                            {
-                                "type": "image_url",
-                                "image_url": {"url": image_data},
-                            },
-                        ],
-                    }
-                ],
-                max_tokens=300,
+                    {"role": "user", "content": f"What's in this image? {image_data}"}
+                ]
             )
-
-            return response.choices[0].message.content
-
+            return response
```

Comment:
I guess response is not a string instance, right?

```suggestion
        return response.choices[0].message.content
```

Reply:
good catch